src/test/java/ArchiveBB/Tests/Events.java

```diff
@@ -0,0 +1,92 @@
+package ArchiveBB.Tests;
+
+import PageObjects.EventsList;
+import helper.Class.InitClass;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.Reporter;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+import videoHelper.VideoRecorder;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.List;
+
+public class Events extends InitClass{
+
+    private VideoRecorder videoRecord;
+    private EventsList eventsList;
+
+    @BeforeMethod
+    public void beforeMethod() {
+        videoRecord = new VideoRecorder(driver);
+        eventsList = new EventsList(driver);
+    }
+
+    @Test()
+    @Parameters({"link", "videoPath"})
+    public void testEventsListEng(String link, String videoPath) throws Exception {
+        // start video recording
+        videoRecord.startRecording(videoPath);
+
+        // do test
+        try {
+            driver.get(link);
+
+            setLang("en");
+            List<WebElement> events = driver.findElements(By.cssSelector(".top.aligned"));
```

Comment:
Test method should include the clean logic only, without define webelements into test body.
Reporter should be implemented inside the page object class, just remember that test body should stay clean :)